lib/OpenCloud/Common/Resource/PersistentResource.php

```diff
@@ -265,7 +276,8 @@ protected function recursivelyAliasPropertyValue($propertyValue)
                 }
             }
         } elseif (is_object($propertyValue) && ($propertyValue instanceof \stdClass)) {
-            foreach ($propertyValue as $key => $subValue) {
+            $objectVars = get_object_vars($propertyValue);
+            foreach ($objectVars as $key => $subValue) {
```

Comment:
Do we need to initialise as a variable? I think we can get away with having foreach call `get_object_vars($propertyValue)` directly